.github/workflows/release.yml

```
@@ -189,144 +154,43 @@ jobs:
-            ${{ steps.stage.outputs.artifact_dir }}
-            dist/${{ needs.metadata.outputs.bin_name }}-${{
-              needs.metadata.outputs.version }}-${{
-              matrix.artifact_suffix }}.pkg
+    uses: ./.github/workflows/build-and-package.yml
+    with:
+      platform: macos
+      runner: ${{ matrix.runner }}
+      target: ${{ matrix.target }}
+      bin-name: ${{ needs.metadata.outputs.bin_name }}
+      version: ${{ needs.metadata.outputs.version }}
+      artifact-name: ${{ env.REPO_NAME }}-${{ matrix.artifact_suffix }}
```

actionlint (1.7.8)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)